### DISCUSSION TITLE
Use semver for Docker images

### DISCUSSION THREAD

**Discussion by danepowell** [grafana/grafana#45342] [General]
I wish Grafana would use semantic versioning for its Docker images (grafana/grafana-oss especially). This would allow me to specify an image such as `grafana/grafana-oss:8` in Docker Compose and automatically receive non-breaking updates in the Grafana 8 branch.

I'm curious what people consider current best practice for keeping up to date using Docker Compose. It seems like you either must use the `main` tag, which means Grafana could jump a major version overnight (yikes), or you have to manually fiddle with docker-compose.yml every time there's a patch release (yuck). I don't like choosing between 'yikes' and 'yuck'! 😄 

I use Watchtower to keep Docker images updated, FWIW.  Here's a [great article explaining the implementation of semver for Docker images](https://medium.com/@mccode/using-semantic-versioning-for-docker-image-tags-dfde8be06699) and why it's helpful.

**Comment by fl0wx** [comment]
What about: https://github.com/grafana/grafana/discussions/47177#discussioncomment-3157311 ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
